symphony/bdk/core/auth/auth_session.py

```diff
@@ -97,4 +97,4 @@ async def session_token(self):
 
     @property
     async def key_manager_token(self):
-        return None
+        return ""
```

Comment:
Having `None` was triggering an error on the generated rest client side.

Reply:
As we "control" the generated client, is it something we could handle?

Reply:
Error is raised here: https://github.com/SymphonyPlatformSolutions/symphony-api-client-python/blob/2.0/symphony/bdk/gen/model_utils.py#L1276
Since it seems complicated to make the correct checks without breaking anything, I would prefer to keep this as is. The generator in java is a bit different: contrary to the python generated api, the java one checks if the tokens are null before putting them as headers.

Reply:
ok